Lib/traceback.py

```diff
@@ -90,7 +91,20 @@ def print_exception(etype, value, tb, limit=None, file=None, chain=True):
     appropriate format, it prints the line where the syntax error
     occurred with a caret on the next line indicating the approximate
     position of the error.
+
+    The *etype* parameter is ignored since python 3.5 and get inferred from
+    *value*.
+
+    Since python 3.7, instead of passing *etype*, *value* and *tb* separately,
+    it is recommended to use the *exc* keyword argument which will infer all the
+    required values.
```

Comment:
You should also document that the traceback object is now get from exc if tb is not set.